src/site/content/en/blog/origin-trials/index.md

```diff
@@ -65,12 +65,17 @@ Find out more: [What are third-party origin trials?](/third-party-origin-trials)
 1. When your token expires, you will get an email with a renewal link.
    To do so, you are again asked to submit feedback.
    
-{% Aside 'warning' %}
-Even if an API lands unchanged after a successful origin trial, there is a short period between the 
+{% Aside %}
+Usually if an API lands unchanged after a successful origin trial, there is a short period between the 
 end of the origin trial and the date the implementation ships in the browser when the API will not 
 be available. This is by design. If Chrome were to avoid the mandatory total-breakage period, that would 
 bias toward also avoiding breakages in the API surface, which are often needed to improve the API.
 The final shipping API might be worse for it.
+
+Exceptionally, if there was clear evidence that developers engaged with the origin trial and that their
```

Comment:
```suggestion
In rare circumstances, if there was clear evidence that developers engaged with the origin trial and that their
```

Reply:
I know it's more words, but 'exceptionally' doesn't quite seem on the mark for what you're conveying.

Reply:
Agree.